tests/e2e/README.md

```diff
@@ -0,0 +1,199 @@
+
+---
+
+## 🚀 Getting Started
+
+### Prerequisites
+- Node.js (LTS version recommended)
+- NPM
+
+### Installation
+1. Navigate to the e2e directory:
+   ```bash
+   cd thunder/tests/e2e
+   ```
+2. Install dependencies:
+   ```bash
+   npm ci
+   ```
+3. Install Playwright browsers:
+   ```bash
+   npx playwright install --with-deps
+   ```
+
+### Configuration
+Create a `.env` file in the `e2e` directory (copy from `.env.example` if available):
+
+```env
+BASE_URL=https://localhost:8090
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=admin
+TEST_USER_USERNAME=testuser
+```
+
+---
+
+## 🏃‍♂️ Running Tests
+
+### Standard Commands
+
+| Command | Description |
+|---------|-------------|
+| `npm test` | Run all tests on all configured browsers (Chromium, Firefox, WebKit) |
+| `npm run test:chromium` | Run tests on Google Chrome/Chromium only |
+| `npm run test:firefox` | Run tests on Firefox only |
+| `npm run test:webkit` | Run tests on Safari/WebKit only |
+| `npm run ui` | Open Playwright's interactive UI mode (Debugging) |
+| `npm run report` | Open the HTML test report |
+
+### Report Merging
+When running tests in parallel or CI, reports are generated as blobs. To merge them into a single HTML report:
+```bash
+npm run posttest
+```
+
+---
+
+## 🏗️ Architecture & Design
+
+### Authentication Flow
+We use Playwright's **Project Dependencies** pattern. The `setup` project runs first, authenticates the user, and saves the session state (cookies, localStorage, sessionStorage) to a JSON file. Subsequent test projects import this state.
+
+```mermaid
+sequenceDiagram
+    participant Runner as Test Runner
+    participant Setup as Setup Project (auth.setup.ts)
+    participant AuthFile as .auth/devportal-admin.json
+    participant Test as Test Project (Chromium/WebKit)
+    participant App as Developer Portal
+
+    Note over Runner, App: Phase 1: Authentication Setup
+    Runner->>Setup: Run auth.setup.ts
+    Setup->>App: Perform Login (UI Interaction)
+    App-->>Setup: Session Tokens (OIDC/OAuth2)
+    Setup->>AuthFile: Save Storage State (JSON)
+    
+    Note over Runner, App: Phase 2: Test Execution
+    Runner->>Test: Run *.spec.ts
+    Test->>AuthFile: Load Storage State
+    Test->>App: Inject Tokens & Navigate
+    Note right of App: User is already logged in!
+    Test->>App: Execute Test Actions (POM)
+```
+
+### Directory Structure
+
+```plaintext
+tests/e2e/
+├── .github/              # GitHub Actions Workflow (moved to repo root)
+├── configs/
+│   ├── routes/           # UI Route definitions
+│   └── api/              # API Route definitions
+├── constants/            # Static constants
+│   ├── http-status.ts    # HTTP Status codes
+│   └── ui-messages.ts    # UI text/labels
+├── data/                 # Test Data Factories
+│   └── test-data.ts      # Dynamic data generators
+├── fixtures/             # Playwright Fixtures
+│   ├── developer-portal-auth.fixture.ts
+│   ├── developer-portal-pom.fixture.ts
+│   └── index.ts
```

Comment:
Dpn't we have to put these into a sub directory
Ex:
- fixtures/
    - developer-portal/
        - developer-portal-auth.fixture.ts
        - developer-portal-pom.fixture.ts

Reply:
Updated